target/scripts/helpers/setup-rspamd.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+# Just a helper to prepend the log messages with `(Rspamd setup)` so
+# users know exactly where the message originated from.
+function __rspamd__log { _log "${1:-}" "(Rspamd setup) ${2}" ; }
+
+# Run miscellaneous checks against the current configuration so we can
+# properly handle integration into ClamAV, etc.
+#
+# This will also check whether Amavis is enabled and emit a warning as
+# we discourage users from running Amavis & Rspamd at the same time.
+function __rspamd__preflight_checks
+{
+  touch /var/lib/rspamd/stats.ucl
+
+  if [[ ${ENABLE_AMAVIS} -eq 1 ]] || [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]]
+  then
+    __rspamd__log 'warn' 'Running Amavis/SA & Rspamd at the same time is discouraged'
+  fi
+
+  if [[ ${ENABLE_CLAMAV} -eq 1 ]]
+  then
+    __rspamd__log 'debug' 'Enabling ClamAV integration'
+    sedfile -i -E 's|^(enabled).*|\1 = true;|g' /etc/rspamd/local.d/antivirus.conf
+    # RSpamd uses ClamAV's UNIX socket, and to be able to read it, it must be in the same group
+    usermod -a -G clamav _rspamd
```

Comment:
Should this not be handled the same way Amavis does in the Dockerfile? (_or the Amavis handling moved to it's setup script for enabling?_)

Reply:
I'd rather do it here since we only really need it when using ClamAV. I'd advocate for moving the Amavis-handling into scripts then (follow-up PR).